Adyen/Utilities/Debugging/FatalErrorUtil.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// :nodoc:
+/// Used for testing code that causes fatalError
+public func fatalError(
```

Comment:
Will this override standard Swift's `fatalError` for everyone who included `AdyenCore` module?

Reply:
`Assert` won't work in this case. If we want to stop execution in an `init` - it has to be a `fatalError`.  
Also good catch, I will make this `internal`, so it does not override `fatalError` for users of the SDK.